coralibre-sdk/testapp/src/main/java/org/schabi/testapp/MainActivity.kt

```diff
@@ -0,0 +1,46 @@
+package org.schabi.testapp
+
+import android.Manifest
+import android.app.AlarmManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Button
+import androidx.appcompat.app.AppCompatActivity
+import org.coralibre.android.sdk.PPCP
+import org.coralibre.android.sdk.internal.TracingService
+import org.coralibre.android.sdk.internal.TracingService.*
+import org.coralibre.android.sdk.internal.TracingServiceBroadcastReceiver
+
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        requestPermissions(arrayOf(Manifest.permission.ACCESS_COARSE_LOCATION, Manifest.permission.ACCESS_FINE_LOCATION), 0);
```

Comment:
So this should now only request `ACCESS_COARSE_LOCATION`

Reply:
done